src/GeneralStateTestsFiller/stEWASMTests/utils/createFromTransactionFiller.yml

```diff
@@ -0,0 +1,55 @@
+createFromTransaction:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    # tx sender
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: ''
+      nonce: ''
+      storage: {}
+  expect:
+    - indexes:
+        data: -1
+        gas: -1
+        value: -1
+      network:
+        - ALL
+      result:
+        # newly-created contract
+        6295ee1b4f6dd65047762f924ecd367c17eabf8f:
+          code: '0x01'
+  transaction:
+    data: 
+    - |
+        (module
+          (import "ethereum" "return" (func $return (param i32 i32)))
+          (import "debug" "printMemHex" (func $printMemHex (param i32 i32)))
+          (memory 1)
+          (export "main" (func $main))
+          (func $main
+            ;; Store some data to return
+            (i32.store (i32.const 0) (i32.const 1))
```

Comment:
Can use `i32.store8` to ensure it is written as 1 byte only. (The current code works since it is little endian, but it is more explicit with `store8`.)